### DISCUSSION TITLE
action Queued

### DISCUSSION THREAD

**Discussion by jinjinzhou** [community/community#51378] [Actions]
### Select Topic Area

Bug

### Body

I cannot commit,  all actions are Queued. 

https://github.com/ucla-biostat-200c/2023spring/actions 

<img width="1324" alt="Screenshot 2023-03-29 at 10 08 12 AM" src="https://user-images.githubusercontent.com/13785167/228615718-f0144817-31dc-45da-a872-435904589dd0.png">

**Comment by Akash1134** [comment]
Hi @jinjinzhou , There is currently an [ongoing incident](https://www.githubstatus.com/incidents/52z0j6phhnjs) with some GitHub Actions. 

[Current Status](https://www.githubstatus.com/): "Actions workflows have started running again. We're continuing to work towards full recovery."

We update our [status page](https://www.githubstatus.com/) in case of an ongoing issue or incident. If you see an open incident, this means that we are looking into it already. If not, please ask the [GitHub community](https://github.com/orgs/community/discussions) or contact [Support](https://support.github.com/contact).

**Reply by umutkacar** [reply]
Hi @Akash1134 , the incident seems to have resolved on GitHub status page, but some of our workflows still stuck on queued state, and we cannot cancel. Also, `gh run` commands return contradicting outputs:

`gh run view 4555170835` displays our workflow as pending
`gh run view --job=4555170835` says `failed to get job: HTTP 404: Not Found (https://api.github.com/repos/deliveryhero/sc-backend-api-py/actions/jobs/4555170835)`
`gh run cancel 4555170835` returns `Cannot cancel a workflow run that is completed`
